circle.yml

```diff
@@ -19,8 +19,7 @@ deployment:
 
 test:
   override:
-    - bundle exec rake knapsack:minitest:
-        parallel: true
+    - bundle exec rails test
```

Comment:
Just a note to us that we should restore parallelization before we declare this good to go. But definitely totally fine until we figure out that weird knapsack error.